src/NServiceBus.Transport.Msmq.Tests/InstanceMapping/InstanceMappingFileMonitorTests.cs

```diff
@@ -70,7 +70,7 @@ public void Reload_should_throw_when_file_does_not_exist()
             });
 
             var monitor = new InstanceMappingFileMonitor("unused", TimeSpan.Zero, timer, fileAccess, new EndpointInstances());
-            await monitor.PerformStartup(null);
+            await monitor.Start(null);
```

Comment:
@Particular/nservicebus-maintainers `PerformStartup` is internal, so I had to add a method to the feature to be able to get the test to work. This seems like on OK workaround, and I'm not thinking we need to do anything here.